src/docs/references/events/index.md

```diff
@@ -401,14 +407,15 @@ Console. Typically this can happen when the event was caused internally
 by an administrative action of the Flex team, in which case the `source`
 of the event would be `source/admin`.
 
-## Transaction process actions and booking and review events
+## Transaction process actions and booking, stock reservation and review events
 
-In Flex, bookings and reviews are primarily managed through the
-transaction process using the
-[booking-](/references/transaction-process-actions/#bookings) and
-[review-related](/references/transaction-process-actions/#reviews)
-actions. Flex emits events separately when each booking or review action
-takes effect, even if multiple actions occur within the same transaction
+In Flex, bookings, stock reservations and reviews are primarily managed
+through the transaction process using the
+[booking-](/references/transaction-process-actions/#bookings),
```

Comment:
What is the "-" after _booking_ (and _stock reservations_) for? Just curious.

Reply:
Someone can correct me, but I think that's the way in english to avoid repeating the part after dash. So instead of "booking-related, stock reservation-related and review-related actions" you write "booking-, stock reservation- and review-related actions". But if it's confusing, probably can be rephrased somehow.

Reply:
Ah, right, got it. Well, I don't know enough about this to correct or not. And it probably doesn't matter too much so no need to continue that discussion or make any update, I was just curious.